other-examples/collector/host-monitoring-nr-infra/README.md

```diff
@@ -0,0 +1,42 @@
+# Monitoring APM with OpenTelemetry Collector and Host monitoring Host with Newrelic infra agent
+
+This example demonstrates Monitoring APM with OpenTelemetry Collector and monitoring hosts with the [Newrelic infra agent](https://docs.newrelic.com/docs/infrastructure/infrastructure-agent/linux-installation/package-manager-install).
```

Comment:
Hold off on changes to this while we work through other parts of the example, but the phrase "This example demonstrates Monitoring APM with OpenTelemetry Collector" isn't right. We've got an APM application monitored with the OpenTelemetry Java Agent, exporting data to an OpenTelemetry collector which is enriching the telemetry with contextual information about the ec2 environment its running in.